index.html

```diff
@@ -54,10 +54,10 @@ <h1>
                 <img src="usedresources/cropped_IMG_2512.png" class="me" alt="Photo of Marissa">
             </div>
                 
-        </div> 
+        </section> 
         
-        <div id="projects">
-            <h2>Projects</h2>
+        <section class="projects">
```

Comment:
Could probably leave the 'id' here on the section, if we're just using it as a scroll target. No need for the extra 'span' inside the 'h2'
```suggestion
        <section id="projects" class="projects">
```

Reply:
Honestly, I don't really know why I added that span! But the id is back on the section tag now.